deploy/cdk/marble-elasticsearch-pipeline/app.py

```diff
@@ -2,15 +2,9 @@
 
 from aws_cdk import core
 from marble_elasticsearch_pipeline.marble_elasticsearch_pipeline_stack import MarbleElasticsearchPipelineStack
-import json
 
-with open('repo.json') as repo_file:
-    repo = json.load(repo_file)
-    repo_name = repo.get('repo_name', 'marble-elasticsearch')
-    repo_branch = repo.get('repo_branch', 'master')
-    repo_owner = repo.get('repo_owner', 'ndlib')
 
 app = core.App()
-meps = MarbleElasticsearchPipelineStack(app, "marble-elasticsearch-pipeline",
-                                        repo_branch, repo_name, repo_owner)
+meps = MarbleElasticsearchPipelineStack(app, "marble-elasticsearch-pipeline")
 meps.add_stages()
+app.synth()
```

Comment:
Is this required?

Reply:
For now. This will be going away in a future cdk update.